text/0000-mixed-shadow-mode.md

```diff
@@ -0,0 +1,175 @@
+inverse is not allowed. Not only does this make things easier to reason about, but many existing
+components rely on workarounds in synthetic mode that are not possible to support in native mode.
+This is further discussed below as observable differences.
+
+### Accessibility
+
+The most common accessibility issue in LWC is related to id-referencing across shadow boundaries.
+The current workaround for this is to bypass the LWC engine's id-mangling by setting attributes
+dynamically, but such a workaround would not work in native Shadow DOM.
+
+### Instrumentation
+
+Applications that rely on instrumentation libraries that don't yet support Shadow DOM are currently
+able to obtain references to descriptors like `addEventListener()` before they are patched and use
+them to override `@lwc/synthetic-shadow` polyfills where needed. Such workarounds would not work for
+events originating from components that prefer native Shadow DOM.
+
+### Styling
+
+Applications currently rely on global stylesheets to apply deeply throughout the DOM. With native
+Shadow DOM, this would not be possible. All of a component's styles would have to be generated and
+added to the component bundle.
+
+### Testing
+
+LWC currently has integration tests for synthetic shadow mode and native shadow mode. A third
+testing mode called "mixed shadow mode" will be introduced. Synthetic shadow DOM polyfills are
+implemented as patches on global prototypes so the decision of invoking a polyfilled API vs a native
+API will depend on the preference of individual components during runtime. This test environment
+will ensure that components designed to run in a native Shadow DOM environment will work correctly
+when even when synthetic Shadow DOM polyfills are present.
+
+Components that prefer native mode will need to also support synthetic mode in browser environments
+where Shadow DOM is natively unavailable, so they will also need to be tested in both modes. LWC
+test utilities will need to be updated to facilitate this.
+
+The existing WPT (Web Platform Tests) test suite for Shadow DOM APIs will also be run to identify
+any coverage gaps in LWC integration tests.
+
+## Observable differences
+
+### Light DOM and assigned elements
+
+In native Shadow DOM, elements are rendered in a component's light DOM and assigned to slots in a
+child component's shadow DOM. This means that elements will exist in the DOM regardless of whether
+they are assigned to a slot. This is not the case for LWC--elements that are passed down to child
+components are never rendered unless they are assigned to a slot.
+
+In the following example, `span` will not exist in the DOM in synthetic shadow, but it will exist in
+native shadow.
+
+```html
+<!-- x-parent -->
+<template>
+    <x-child>
+        <span>foo</span>
+    </x-child>
+</template>
+
+<!-- x-child -->
+<template>
+    <p>child</p>
+</template>
+```
+
+This observable difference will not be remediated as such a change would be non-trivial and

```

Comment:
Another difference worth mentioning is timing, in the case where the slotted contents are ordered differently from the slots as defined in the slottable component. (See [timing comparison](https://github.com/salesforce/lwc-rfcs/blob/a3f6d7182de687554ef175b1228b0c9e06271619/text/0000-light-dom.md#slots))

Reply:
Great point, thanks!